utils.py

```diff
@@ -319,8 +319,6 @@ def bootstrap_nodes(account_ids, chain, **custom_flags):
         flags = {
             '--chain-id': 'a0tnet1',
             '--chain-name': 'AlephZeroTestnet',
-            '--session-period': '900',
```

Comment:
you should also remove these flags from `docker-compose`

Reply:
Right, good point.